example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -138,6 +149,99 @@ describe('The Lzards Backup Task ', () => {
     expect(backupStatus[0].statusCode).toBe(201);
   });
 
+  describe('The Lzards API Client', () => {
```

Comment:
Nice, I like the single-lambda/component invocation integration test approach here, generally.   :100: